docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -23,7 +23,29 @@ If the user name is `Alice` and the password is `secret`, the HTTP authorization
 ====
  
 The Basic Authentication mechanism does not provide confidentiality protection for the transmitted credentials. The credentials are merely encoded with Base64 when in transit and not encrypted or hashed in any way. Therefore, Basic Authentication is used with HTTPS to provide confidentiality.
- 
+
+Basic Authentication is a well-specified, simple challenge and response scheme that all web browsers and most web servers understand. However, there are a few limitations associated with Basic Authentication, which include:
+
+Credentials are sent as plain text::
++
+--
+It is required to use HTTPS with Basic Authentication to avoid exposing the credentials. However, if the load balancer terminates HTTPS, the risk of exposing credentials as plain text increases when a request is forwarded to Quarkus over HTTP.
+
+Also, in multi-hop deployments, the credentials can be exposed if HTTPS is used between the client and the first Quarkus endpoint only. However, the credentials are propagated to the next Quarkus endpoint over HTTP.
```

Comment:
```suggestion
Also, in multi-hop deployments, the credentials can be exposed if HTTPS is used between the client and the first Quarkus endpoint only and the credentials are propagated to the next Quarkus endpoint over HTTP.
```

Reply:
@sberyozkin Thanks for adding that. Fixed the sentence as suggested.